examples/interactive/cli-platform/Env.roc

```diff
@@ -13,20 +14,32 @@ cwd =
 
     InternalTask.fromEffect effect
 
-# ## Sets the [current working directory](https://en.wikipedia.org/wiki/Working_directory)
-# ## in the environment. After changing it, file operations on relative [Path]s will be relative
-# ## to this directory.
-# setCwd : Path -> Task {} [InvalidCwd]* [Env]*
-# setCwd = InternalTask.fromEffect Effect.setCwd
-# ## Gets the path to the currently-running executable.
-# exePath : Task Path [ExePathUnavailable]* [Env]*
-# exePath = InternalTask.fromEffect Effect.setCwd
+## Sets the [current working directory](https://en.wikipedia.org/wiki/Working_directory)
+## in the environment. After changing it, file operations on relative [Path]s will be relative
+## to this directory.
+setCwd : Path -> Task {} [InvalidCwd]* [Read [Env]*]*
```

Comment:
Since this is modifying the environment, should it be

```suggestion
setCwd : Path -> Task {} [InvalidCwd]* [Write [Env]*]*
```

?

Reply:
Yup, nice catch!